src/flashbar/__integ__/collapsible.test.ts

```diff
@@ -2,28 +2,29 @@
 // SPDX-License-Identifier: Apache-2.0
 import { setupTest } from './pages/interactive-page';
 import { FOCUS_THROTTLE_DELAY } from '../utils';
+import { wait } from '../../internal/utils/promises';
```

Comment:
This utility already exists, `page.pause()`

Same may also argue that `page.waitForAssertion()` may work better for your case

Reply:
Had a similar discussion here: https://github.com/cloudscape-design/components/pull/642#discussion_r1090366256

I prefer to wait exactly this amount of time because I actually _know_ I have to wait for the focus throttle delay (related to the other discussion below in this PR).

But I didn't know about `page.pause`, makes sense.